docs/fundamentals/executor/hub/use-hub-executor.md

```diff
@@ -37,10 +37,10 @@ Use prebuilt images from Hub in your Python code:
 from jina import Flow
 
 # SECRET must be provided for private Executor
```

Comment:
let's also handle a case for private executors. Maybe let's say now we have to login to jina hub ?